regolith/schemas.py

```diff
@@ -231,6 +231,14 @@
             for i in range(1, 11)
         ],
         "payee": "scopatz",
+        "reimbursements": [
```

Comment:
we should probably think about having a test for a non tbd reimbursement to cover everything, but decide if that is too much work and is worth it.

Reply:
I added a non tbd reimbursement to test out